src/app/components/AppNavigator/index.tsx

```diff
@@ -27,7 +28,15 @@ const AuthNavigator = createStackNavigator<AuthNavigatorParams>();
 function LibraryStackScreen() {
   return (
     <LibraryNavigator.Navigator initialRouteName={Routes.BookList} screenOptions={stackNavigatorConfig}>
-      <LibraryNavigator.Screen name={Routes.BookList} component={BookList} options={{ title: 'LIBRARY' }} />
+      <LibraryNavigator.Screen
+        name={Routes.BookList}
+        component={BookList}
+        options={{
```

Comment:
I'd move this entire `options` to `@config/navigation`. Same for the other Screens